src/__tests__/ComponentsTesting/App.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
- [ ] Some of your tests are failing kindly fix them!
![Screenshot from 2021-11-25 20-20-59](https://user-images.githubusercontent.com/62705947/143492468-65f46bbe-c88a-40f4-b25c-5c7fe1cd720c.png)